pango/src/auto/font_map.rs

```diff
@@ -19,6 +19,26 @@ impl FontMap {
 }
 
 pub trait FontMapExt: IsA<FontMap> + 'static {
+    #[cfg(feature = "v1_56")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v1_56")))]
+    #[doc(alias = "pango_font_map_add_font_file")]
+    fn add_font_file(&self, filename: &str) -> Result<(), glib::Error> {
```

Comment:
should this be path?

Reply:
Yes, will send an MR. Thanks!

Reply:
Seems like gitlab is broken currently

Reply:
https://gitlab.gnome.org/GNOME/pango/-/merge_requests/775

Reply:
Done